kafka/encoder.go

```diff
@@ -14,13 +14,13 @@ import (
 
 	"github.com/Shopify/sarama"
 	"github.com/aristanetworks/glog"
-	"github.com/golang/protobuf/proto"
+	pb "github.com/openconfig/gnmi/proto/gnmi"
 )
 
 // MessageEncoder is an encoder interface
 // which handles encoding proto.Message to sarama.ProducerMessage
 type MessageEncoder interface {
-	Encode(proto.Message) ([]*sarama.ProducerMessage, error)
```

Comment:
I think we can keep the interface generic the way it was intended, so we don't couple ourselves with the types of messages we're handling.